content/en/integrations/guide/aws-integration-troubleshooting.md

```diff
@@ -59,18 +59,24 @@ When installing the Agent on an AWS host, you might see duplicated hosts on the
 
 ### EC2 metadata with IMDS v2
 
-In some situations, the configuration of EC2's [IMDSv2][5] makes it impossible for the agent to access metadata, leading the Agent to fall back to the `os` hostname provider instead of `aws`, as seen in the output of `agent status`.
-
 In containerized environments the problem might be that you have locked down the EC2 metadata endpoint, by way of assigning IAM roles/credentials to pods running in the Kubernetes cluster. `Kube2IAM` and `kiam` are common tools used to do this. To solve this, update your `Kube2IAM` or `kiam` configuration to allow access to this endpoint.
 
-The AWS API supports disabling IMDSv1, which the Agent uses by default. If this is the case, but IMDSv2 is enabled and accessible, set the parameter `ec2_prefer_imdsv2` to `true` (defaults to `false`) in your [Agent configuration][6]. See the [Transition to using Instance Metadata Service Version 2][7] documentation for details.
-
 IMDSv2, in its default configuration, refuses connections with an IP hop count greater than one, that is, connections that have passed through an IP gateway. This can cause problems when the Agent is running in a container with a network other than the host's network, as the runtime forwards the container's traffic through a virtual IP gateway. This is common in ECS deployments. The following options may remedy this issue:
 
- * [Increase the maximum hop count to at least `2`][8]. Doing so may have implications for the security of data stored in the IMDS, as it permits containers other than the Agent to access this data as well. 
+ * [Increase the maximum hop count to at least `2`][8]. Doing so may have implications for the security of data stored in the IMDS, as it permits containers other than the Agent to access this data as well.
  * Use the hostname discovered by cloud-init, by [setting `providers.eks.ec2.useHostnameFromFile` to true][9].
  * Run the Agent in the host UTS namespace, by [setting `agents.useHostNetwork` to true][10].
 
+### EC2 hostname with IMDS
+
+#### Before Agent release 7.64.0
+
+In some situations, the configuration of EC2's [IMDSv2][5] makes it impossible for the Agent to access metadata. This causes the Agent to fall back to the `os` hostname provider instead of `aws`, as seen in the output of `agent status`.
```

Comment:
```suggestion
In some situations, the of EC2 [IMDSv2][5] configuration may make it impossible for the Agent to access the necessary metadata. This can cause the Agent to fall back to the `os` hostname provider instead of `aws`, as seen in the output of `agent status`.
```